app/src/main/java/com/bignerdranch/android/playlistmaker/search/ui/fragment/TrackViewHolder.kt

```diff
@@ -28,7 +28,7 @@ class TrackViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
         Glide.with(itemView)
             .load(track.artworkUrl100)
             .placeholder(R.drawable.no_replay)
-            .transform(RoundedCorners(4))
+            .transform(RoundedCorners(8))
             .into(imageViewArtwork)
```

Comment:
🍏 Сейчас обложки треков у тебя слабо закруглены, потому что ты передаешь в RoundCorners значение в  dp, а тебе сюда нужно передать количество пикселей, которое будет разным в зависимости от экрана. Попробуй следующий подход:
```suggestion
        Glide.with(itemView)
            .load(track.artworkUrl100)
            .placeholder(R.drawable.no_replay)
            .transform(
                RoundedCorners(
                         itemView.resources.getDimensionPixelSize(R.dimen.album_cover_corner_radius)
                )
            )
            .into(imageViewArtwork)

```